### PR TITLE
change folder hierarchy

### PR DESCRIPTION
**Pull request by core-man**
Remove subfolder seismic-wave
In the url, taup is directly after software﻿ now

### CODE REVIEW

File: source/ray-theory.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+射线理论
```
Comment (seisman) [inline source/ray-theory.rst:1]: I don't think we need this file.

File: source/index.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@
    :caption: 地震波
    :hidden:
 
-   seismic-wave/index
+   ray-theory
```
Comment (seisman) [inline source/index.rst:10]: I think including `taup` here is enough.
```suggestion
   taup
```

**Comment by seisman** [comment]
BTW, no need to keep the whole `seismic-wave` directory.

File: source/index.rst (hunk 2 above)
Comment (core-man) [reply source/index.rst:10]: If we have too much software, the left side bar would be too long.

File: source/ray-theory.rst (hunk 1 above)
Comment (core-man) [reply source/ray-theory.rst]: done

**Review by seisman — APPROVED** [review]

**Pull request merged**